app/worker/process.go

```diff
@@ -142,25 +144,23 @@ func processTask(task model.Task, job *model.Job,
 				"Error: Task definition incompatible " +
 					"with specified type (http)")
 		}
-		l.Infof("processTask: calling http task: %s %s", httpTask.Method,
-			processJobString(httpTask.URI, workflow, job))
```

Comment:
Why did you remove this log?

Reply:
1. I moved it below
2. it was doubling the very expensive operation of processing the job string, just for printing.